service/controller.go

```diff
@@ -2880,7 +2883,7 @@ func (s *service) GetCapacity(
 	}
 
 	// Storage (resource) Pool. Validate it against exist Pools
-	storagePoolID := params[StoragePoolParam]
+	storagePoolID := s.resolveParameter(params, symmetrixID, StoragePoolParam, "")
 	err = s.validateStoragePoolID(ctx, symmetrixID, storagePoolID, pmaxClient)
 	if err != nil {
```

Comment:
the changes to GetCapacity are good, but to resolve this issue:
```
REP 0073: rpc error: code = InvalidArgument desc = GetCapacity: Required StoragePool and SymID in parameters"
```
This check needs to be removed: https://github.com/dell/csi-powermax/pull/493/files#diff-8120cde4a65d55a72e0ffe83dd202cadcc210138f8b1525e77ac20aa4e0c7388L2862

Reply:
Fixed by removing chekc.